### PR TITLE
Fix #23: decode LaTeX accented characters to Unicode in web API

### PR DESCRIPTION
**Pull request by JulesBlanchard**
Add tex2unicode() to convert LaTeX commands (e.g. {\'e}) to Unicode before returning JSON to ReLiS. Add unit tests covering all accent types and edge cases.

### CODE REVIEW

File: src/bibler/web.py [hunk 1]
```diff
@@ -55,10 +56,17 @@ def entryToJSON(entry, biblerapp):
...
     """
+    utils = Utils()
     json = {}
     json["result_code"] = int(entry[EntryListColumn.Valid])
     json["result_msg"] = entry[EntryListColumn.Message]
-    json["entry"] = entry
+    # Decode LaTeX-encoded characters to Unicode in all text fields
+    # before returning the entry to the caller (fix for issue #23)
+    decoded_entry = dict(entry)
+    for field in ['title', 'abstract', 'school', 'note', 'annote']:
```
Comment (gauranshkumar) [inline src/bibler/web.py]: Just curious why only these fields and not all the fields in the entry?

Comment (JulesBlanchard) [reply src/bibler/web.py]: You're right, the choice was conservative. Those 5 fields are the most common ones containing free text with accented characters. However, fields like booktitle, journal, publisher, address, and series could also contain LaTeX-encoded characters. I can either apply tex2unicode() to all string fields dynamically, or extend the list. Which approach do you prefer?

Comment (JulesBlanchard) [reply src/bibler/web.py]: i think all string fields dynamically is better

**Review by gauranshkumar — APPROVED** [review]

**Comment by gauranshkumar** [comment]
@JulesBlanchard, I noticed an issue with the package you used, which caused the CI to fail. I checked PyPi https://pypi.org/project/pylatexenc/, and it is at 2.10, and all the latest builds are alpha for 3.* So I am not sure if we can use this package. Can you please look into it and create another PR? 

Thanks!
